android/src/main/java/com/dieam/reactnativepushnotification/modules/RNPushNotificationHelper.java

```diff
@@ -169,31 +169,37 @@ public void sendToNotificationCentre(Bundle bundle) {
             String message = bundle.getString("message");
             String bundle_title = bundle.getString("bundle_title");
             String bundle_id = bundle.getString("bundle_id");
-            int bundle_id_int = Integer.parseInt(bundle_id);
 
-            Notification newMessageNotification1 =
+            NotificationCompat.Builder newMessageNotification1 =
                     new NotificationCompat.Builder(context, NOTIFICATION_CHANNEL_ID)
                             .setSmallIcon(smallIconResId)
                             .setContentTitle(title)
-                            .setContentText(message)
-                            .setGroup(bundle_id)
-                            .build();
-
-            Notification summaryNotification =
-                    new NotificationCompat.Builder(context, NOTIFICATION_CHANNEL_ID)
-                            .setSmallIcon(smallIconResId)
-                            .setStyle(new NotificationCompat.InboxStyle()
-                                    .setSummaryText(bundle_title))
-                            .setGroup(bundle_id)
-                            .setGroupSummary(true)
-                            .build();
-
-            NotificationManager notificationManager = notificationManager();
-
-            checkOrCreateChannel(notificationManager);
-
-            notificationManager.notify(new Random().nextInt(), newMessageNotification1);
-            notificationManager.notify(bundle_id_int, summaryNotification);
+                            .setContentText(message);
+
+            if(bundle_title != null && bundle_id != null){
+                //LP: is supposed to be grouped message
+                int bundle_id_int = Integer.parseInt(bundle_id);
+
+                newMessageNotification1.setGroup(bundle_id);
+
+                NotificationCompat.Builder summaryNotification =
+                        new NotificationCompat.Builder(context, NOTIFICATION_CHANNEL_ID)
+                                .setSmallIcon(smallIconResId)
+                                .setStyle(new NotificationCompat.InboxStyle()
+                                        .setSummaryText(bundle_title))
+                                .setGroup(bundle_id)
+                                .setGroupSummary(true);
+
+                NotificationManager notificationManager = notificationManager();
+                checkOrCreateChannel(notificationManager);
+                notificationManager.notify(new Random().nextInt(), newMessageNotification1.build());
```

Comment:
can you have a look and see how the library creates the ids? and possibly od the same?

I think it was using some sort of ID from google. Which would be better just in case you get a duplicate.